buildchain/buildchain/targets/local_image.py

```diff
@@ -84,22 +84,24 @@ def show(_task: types.Task) -> str:
 
     def _build_actions(self) -> List[types.Action]:
         """Build a container image locally."""
-        build_cmd = [
-            config.DOCKER, 'build',
-            '--tag', self.tag,
-            '--file', self.dockerfile,
-        ]
-        for arg, value in self.build_args.items():
-            build_cmd.extend(['--build-arg', '{}={}'.format(arg, value)])
-        build_cmd.append(self.dockerfile.parent)
-
-        actions : List[types.Action] = [build_cmd]
+        actions : List[types.Action] = []
+
+        # actions.append(self.check_dockerfile_dependencies)
+
+        builder = BuildCommand(
+            tag=self.tag,
+            path=self.dockerfile.parent,
+            dockerfile=self.dockerfile,
+            args=self.build_args
+        )
+        actions.append((builder, [], {}))
+
         # If a destination is defined, let's save the image there.
         if self.save_on_disk:
             filepath = self.uncompressed_filename
-            actions.append([
-                config.DOCKER, 'save', self.tag, '-o', str(filepath)
-            ])
+            actions.append(
+                (self.save_image, [], {'save_path': str(filepath)})
```

Comment:
Don't need to cast to `str` here, you defined `save_image(self, save_path: Path)` so you expect a `Path` here.